views/attendance/attendance_form.php

```diff
@@ -20,11 +20,15 @@
 
 include ('../models/Notification.php');
 
-require "attendance_utilities.php";
+require ("attendance_utilities.php");
+require ("shared_queries.php");
 
 //if attendance has already started being recorded, grab that information from the session information
 $pageInformation = isset($_SESSION['serializedInfo']) ? deserializeParticipantMatrix($_SESSION['serializedInfo']) : array();
 
+//participant's number to edit from edit participant page
+if(isset($_SESSION['edit-participant-details-num'])) unset($_SESSION['edit-participant-details-num']);
```

Comment:
Put the body of the if-statement on a separate line so it is easier to read.

Reply:
done